### ISSUE TITLE
The connection to the terminal's pty host process is unresponsive, the terminals may stop working

### ISSUE THREAD

**Issue by Nisarg01-01** [microsoft/vscode#121369]
Version: 1.55.2 (user setup)
OS: Windows_NT x64 10.0.19042

**Comment by Nisarg01-01** [comment]
Version: 1.55.2 (user setup)
Commit: 3c4e3df9e89829dce27b7b5c24508306b151f30d
Date: 2021-04-13T09:35:57.887Z
Electron: 11.3.0
Chrome: 87.0.4280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Windows_NT x64 10.0.19042

**Comment by Nisarg01-01** [comment]
I am new to Vscode so I am not Familiar with many terms like Insider build and other thing please explain what to do?

**Comment by meganrogge** [comment]
I believe this is already fixed, can you see if you can reproduce in the [Insiders build](https://code.visualstudio.com/insiders)?

**Comment by Nisarg01-01** [comment]
What does is means by reproducing insider build

**Comment by markserlin-crucible** [comment]
I am seeing this issue repeatedly 
Version: 1.55.2 (user setup)
Commit: 3c4e3df9e89829dce27b7b5c24508306b151f30d
Date: 2021-04-13T09:35:57.887Z
Electron: 11.3.0
Chrome: 87.0.4280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Windows_NT x64 10.0.19041

When > 1 terminal window is open either side by side or in separate mode, or indeed on sepearte instances of VSCode.
PS I use the Terminal confgured as  a git bash

**Comment by khorne07** [comment]
Same issue when running multiple terminal instances in version 1.55.2 on Windows 10.

**Comment by markserlin-crucible** [comment]
So this thread has been closed, and the one referred to as duplicate above is also closed. I smell BS.

**Comment by markserlin-crucible** [comment]
I am seeing this issue repeatedly
Version: 1.55.2 (user setup)
Commit: 3c4e3df
Date: 2021-04-13T09:35:57.887Z
Electron: 11.3.0
Chrome: 87.0.4280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Windows_NT x64 10.0.19041

When > 1 terminal window is open either side by side or in separate mode, or indeed on sepearte instances of VSCode.
PS I use the Terminal confgured as a git bash

I believe issue was closed for 1.54.x but see version above

I tried the latest Insider's version and the Terminal dod not open at all - completely broken.
Steps: Menu->Terminal->New Terminal
No response.
